src/components/auth/profile/DisableAccountModal.tsx

```diff
@@ -37,24 +38,40 @@ const StyledModal = styled(Modal)(({ theme }) => ({
     backgroundColor: '#EEEEEE',
     padding: 20,
     p: 4,
+    whiteSpace: 'pre-line',
     [theme.breakpoints.down('md')]: {
-      width: '70%',
+      width: '90%',
     },
   },
+  [`& .${classes.subtitle}`]: {
+    fontFamily: 'Lato, sans-serif',
+    fontStyle: 'normal',
+    fontWeight: '400',
+    fontSize: theme.typography.pxToRem(14),
+    lineHeight: theme.spacing(3),
+    color: theme.palette.primary.main,
+    margin: `${theme.spacing(1)} 0`,
```

Comment:
We don't need to interpolate the theme values.
I suggest that here you could use margin: theme.spacing(1, 0),

Reply:
Addressed, thank you.